### ISSUE TITLE
Is it possible to remove winpty

### ISSUE THREAD

**Issue by shanjunmei** [runletapp/go-console#21]
does it possible replace with https://github.com/golang/term or https://github.com/containerd/console

**Comment by NHAS** [comment]
No, winpty translates windows console events into pty characters. 

`golang/term` is effectively a terminal emulator, it manages the visual state of a pty by sending control characters, but does not translate windows console events to pty characters. 

`containerd/console` provides methods for controlling ptys on linux, and the console on windows. It does not provide a method for converting it into a pty compatible form.
